www/index.php

```diff
@@ -62,6 +62,10 @@
         $conn['plr'] = 0;
     $connectivity['custom'] = $conn;
 }
+$recaptcha = GetSetting['6LfYXTwUAAAAAPyn1ID4UdsXs-ja3P7qPib37asw'];
+if ($recaptcha) {
+    echo <recaptcha html with loaded 6LfYXTwUAAAAAPyn1ID4UdsXs-ja3P7qPib37asw>

```

Comment:
what is this?

Reply:
6LfYXTwUAAAAAPyn1ID4UdsXs-ja3P7qPib37asw is a api key sorry very much
but i change for hard-coded